README.md

```diff
@@ -1 +1,9 @@
 # GUI-QUIZ
+A simple quiz game using pyhton GUI(Graphical User Interface)-tkinter
+Python offers multiple options for developing GUI (Graphical User Interface). Out of all the GUI methods, tkinter is most commonly used method. It is a standard Python interface to the Tk GUI toolkit shipped with Python. Python with tkinter outputs the fastest and easiest way to create the GUI applications. Creating a GUI using tkinter is an easy task.
```

Comment:
You explained what GUI means on line 2, I don't think you need to explain it here too